src/main/res/values-v21/themes.xml

```diff
@@ -1,26 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<style name="Theme.Amahi" parent="@android:style/Theme.Material">
-		<item name="android:actionBarStyle">@style/Theme.Amahi.ActionBar</item>
+	<style name="Theme.Amahi" parent="@style/Theme.AppCompat.NoActionBar">
 		<item name="android:windowBackground">@color/background_primary</item>
 		<item name="android:colorAccent">@color/accent_material_dark</item>
 		<item name="android:textColorPrimaryInverse">@color/primary_text_material_light</item>
 	</style>
 
-	<style name="Theme.Amahi.Fullscreen.ActionBar" parent="@android:style/Widget.Material.ActionBar">
-		<item name="android:background">@color/background_transparent_primary</item>
-		<item name="android:displayOptions">showHome|homeAsUp|showTitle</item>
-	</style>
-
-	<style name="Theme.Amahi.Fullscreen" parent="@android:style/Theme.Material">
-		<item name="android:actionBarStyle">@style/Theme.Amahi.Fullscreen.ActionBar</item>
+	<style name="Theme.Amahi.Fullscreen" parent="@style/Theme.AppCompat.NoActionBar">
 		<item name="android:windowActionBarOverlay">true</item>
 		<item name="android:windowBackground">@android:color/black</item>
 		<item name="android:windowContentOverlay">@null</item>
 	</style>
 
+	<style name="Theme.Amahi.Settings" parent="@android:style/Theme.Material">
+		<item name="android:actionBarStyle">@style/Theme.Amahi.ActionBar</item>
+		<item name="android:windowBackground">@color/background_primary</item>
+		<item name="android:colorAccent">@color/accent_material_dark</item>
```

Comment:
Android Studio gives a warning here: `The resource @color/accent_material_dark is marked as private in com.android.support:appcompat-v7`.
Please fix this.